src/components/molecules/timeline-events/exchange.tsx

```diff
@@ -112,13 +112,13 @@ const Exchange: React.FC<ExchangeProps> = ({ event, refetch }) => {
     event.exchangeCartId
   )
 
-  const handleCancelExchange = () => {
-    cancelExchange.mutate(event.id)
+  const handleCancelExchange = async () => {
+    await cancelExchange.mutateAsync(event.id)
     refetch()
   }
 
-  const handleCancelReturn = () => {
-    cancelReturn.mutate()
+  const handleCancelReturn = async () => {
+    await cancelReturn.mutateAsync()
```

Comment:
since this is `async` now, `DeletePrompt.onDelete` prop can accept just  `handleCancelReturn` instead of 
`async () => handleCancelReturn()`

Reply:
Good catch 😄